src/input/reader_to_storage.cc

```diff
@@ -69,13 +69,17 @@ ReaderToStorage::ReaderToStorage(const FilePath &in_file, Type::TypeBase &root_t
 		THROW(ExcInputMessage() << EI_Message("Invalid extension of file " + fname + ".\nMust be 'con' or 'yaml'."));
 	}
 
-	std::ifstream in;
-	in_file.open_stream(in);
+	try {
```

Comment:
Find other calls of ReadeToStorage constructor (in particular in make_sub_storage methods) and remove duplicate catches.

Reply:
Done in 9e08f2bfce331359f3b85efd1cea9a52b3ee9e91.